requirements.txt

```diff
@@ -13,6 +13,7 @@ click>=8.0,<9.0
 click-help-colors>=0.9.1,<0.10.0
 tqdm>=4.62,<4.63
 more-itertools>=8.0,<9.0
+petname
```

Comment:
Probably a good idea to pin this, even if it's a very loose pin.

Reply:
f13d717